docs/MAC-Token-Based-Authentication.md

```diff
@@ -1,35 +1,39 @@
 
-This is why PowerAuth also supports a simplified MAC Token Based Authentication. As the name suggests, the authentication is achieved by computing a MAC / digest of a pre-shared token.
+This is why PowerAuth also supports simplified MAC Token-Based Authentication. As the name suggests, the authentication is achieved by computing a MAC (also called a "digest") using a pre-shared token.
 
 ## Security Considerations
 
-There are couple very important things you need to keep in mind while using MAC Token Based Authentication in your APIs:
+There are a couple of very important things to keep in mind while using MAC Token-Based Authentication in your APIs:
 
 - **Data Integrity** - Since the resulting digest does not include any request data, it does not prevent data from being modified.
-- **Replay Attack** - By default, there is no replay attack protection when validating the tokens. You need to implement your own replay attack protection in case you need it, for example by storing already used values of `timestamp` and `nonce` for given `token_id` and disallowing repeated values in certain time window.
-- **Single Factor** - While the token has an information about the factors used while the token was created, which is handy while distinguishing different grades of information (for example, some more sensitive info may require token that was created using 2FA), the authentication as such uses only a single factor. It does not include PIN/password or biometric information at all.
+- **Single Factor** - While the token has information about the factors used while the token was created, which is handy while distinguishing different grades of information (for example, some more sensitive info may require a token that was created using 2FA), the authentication as such uses only a single factor. It does not include PIN/password or biometric information at all.
 
-As a result, you must use MAC Token Based Authentication for read-only operations only. In other words, use the MAC Token Based Authentication to access resources, not to create or modify them. Use 1FA PowerAuth signature for active operations that create or modify resources. This way, you avoid having repeated or inconsistent data, while allowing access to information that needs to be frequently accessed.
+As a result, you must use MAC Token-Based Authentication for read-only operations only. In other words, use the MAC Token-Based Authentication to access resources, not to create or modify them. We recommend using the 1FA PowerAuth signature for active operations that create or modify resources but do not require user information. This way, you avoid having repeated or inconsistent data while allowing access to information that needs to be frequently accessed.
```

Comment:
but do not require user information

I suggest
but do not require to inform a user.
or
but do not require a user's explicit consent.
or 
that can be done on behalf of a user and do not require knowledge or inheritance elements.

Reply:
... this should have been "interaction"...